src/main/java/org/apache/commons/lang3/time/FastTimeZone.java

```diff
@@ -29,14 +29,14 @@
 
     private static final Pattern GMT_PATTERN = Pattern.compile("^(?:(?i)GMT)?([+-])?(\\d\\d?)?(:?(\\d\\d?))?$");
 
-    private static final TimeZone GREENWICH = new GmtTimeZone(false, 0, 0);
+    //private static TimeZone GREENWICH =
 
     /**
      * Gets the GMT TimeZone.
      * @return A TimeZone with a raw offset of zero.
      */
     public static TimeZone getGmtTimeZone() {
-        return GREENWICH;
+        return new GmtTimeZone(false, 0, 0);
```

Comment:
I believe the reason for the static object was to avoid instantiating a new object every time this method was called.